.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+*.svg
+*.jpg
+*.png
```

Comment:
- [ ] Kindly remove the svg, jpg, and png from the .gitignore file as you can push them to the repo so we can view them on the local clone.